device/dhcp/servers/v4/dhcpv4server.yaml

```diff
@@ -0,0 +1,30 @@
+components:
+  schemas:
+    DhcpServer.V4:
```

Comment:
Name would be `DhcpServer.V4Interface` which is align with BGP (`Bgp.V4Interface`). Also this object acts like an interface.